store/tikv/2pc.go

```diff
@@ -430,7 +430,7 @@ func (c *twoPhaseCommitter) commitSingleBatch(bo *Backoffer, batch batchKeys) er
 	// Under this circumstance,  we can not declare the commit is complete (may lead to data lost), nor can we throw
 	// an error (may lead to the duplicated key error when upper level restarts the transaction). Currently the best
 	// solution is to populate this error and let upper layer drop the connection to the corresponding mysql client.
-	isPrimary := bytes.Equal(batch.keys[0], c.primary())
+	isPrimary := bytes.Compare(batch.keys[0], c.primary()) == 0
```

Comment:
why change it?

Reply:
I do not make the change, I will rebase my code.